lib/engine/config/game/g_18_cz.rb

```diff
@@ -169,7 +169,7 @@ module G18CZ
                 "8896": {
                     "count": 1,
                     "color": "red",
-                    "code": "city=revenue:yellow_20|green_30|brown_40|gray_50;path=a:0,b:_0,terminal:1;path=a:1,b:_0,terminal:1;icon=image:18_cz/50"
+                    "code": "city=revenue:yellow_20|green_30|brown_40|gray_50;path=a:0,b:_0,terminal:1;path=a:1,b:_0,terminal:1;icon=image:18_cz/50;label=SX"
```

Comment:
do we want to add requisite labels to the other special offboards as well?
![Screenshot 2021-01-19 at 8 28 33 PM](https://user-images.githubusercontent.com/1711810/105127113-0080cc80-5a95-11eb-9c9d-b97003833416.png)

Reply:
actually I'd suggest we don't label the tiles, since it looks a bit odd when there's also the corp's token on it

![Screenshot 2021-01-19 at 8 30 27 PM](https://user-images.githubusercontent.com/1711810/105127216-3f168700-5a95-11eb-92f0-5d4ca88a7820.png)

Reply:
check this commit out, which adds those labels and removes the need for a constant lookup dict (since we can choose tiles by their label)
https://github.com/benjaminxscott/18xx/commit/b0d98824ce2f61748a698c90a30e731de93d18d6

Reply:
thanks. thats better